package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --config webpack.config.prod.js",
-    "build-dev": "webpack --config webpack.config.dev.js",
     "start": "npm run build && ./leapchat",
-    "test": "node_modules/.bin/mocha  --reporter nyan test/.setup.js test/**/*.test.js"
+    "build-dev": "webpack --config webpack.config.dev.js",
```

Comment:
Well we do have the "dev" script that builds the frontend and runs the backend. It's even shorter :wink: